core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -660,4 +690,44 @@ public RestConfig(ConfigDef definition) {
   public Time getTime() {
     return defaultTime;
   }
+
+  public static void validateHttpResponseHeaderConfig(String config) {
```

Comment:
We're passing this config to Jetty's HeaderFilter. Does that code do this kind of validation? Are we doing this in order to get nicer messages if it fails?

Reply:
Yes. HeaderFilter will validate some configuration. Our code just do more detail checking before pass HeaderFilter.